charts/barman/templates/configmap-pgpass.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
```

Comment:
Given that this file contains some secret info it should probably be a `Secret` and not a `ConfigMap`.